src/osemgrep/language_server/server/RPC_server.mli

```diff
@@ -1,3 +1,8 @@
+module Io : sig
```

Comment:
again, if this is exposed just for testing purpose, say it, and put it at the end of the .mli instead of the start.